textselect.hpp

```diff
@@ -47,6 +47,8 @@ class TextSelect {
     // Indicates whether selection should be updated. This is needed for distinguishing mouse drags that are
     // initiated by clicking the text, or different element.
     bool shouldHandleMouseDown = false;
+    
+    bool enableWordWrap = true;

```

Comment:
We set this field in the constructor initializer list so we won't need a dedicated initializer here. Also, a brief comment about what this field does would be nice for consistency.

```suggestion
    // Indicates whether text selection with word wrapping should be enabled.
    bool enableWordWrap;

```